jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -45,7 +44,6 @@
         (let ((tmp-actions (gethash :larm-controller controller-table)))
           (setf (gethash :larm-controller controller-table) (reverse tmp-actions))))
     ;; initialize slots
-    (setq tfl- (instance ros::transform-listener :init))
```

Comment:
`*tfl*` is set in https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L157-L163